src/transactions/TransactionUtils.cpp

```diff
@@ -331,6 +331,37 @@ acquireLiabilities(AbstractLedgerTxn& ltx, LedgerTxnHeader const& header,
     acquireOrReleaseLiabilities(ltx, header, offer, true);
 }
 
+bool
+addBalanceSkipAuthorization(LedgerTxnHeader const& header,
+                            LedgerTxnEntry& entry, int64_t amount)
+{
+    if (entry.current().data.type() != TRUSTLINE)
```

Comment:
I don't know if you really need this. Trying to access the wrong branch in an xdrpp discriminated union results in an exception anyway.

Reply:
Good point. Removed.